spec/models/druid_version_zip_spec.rb

```diff
@@ -1,8 +1,7 @@
 require 'rails_helper'
 
 describe DruidVersionZip do
-  subject(:dvz) { described_class.new(druid, version) }
```

Comment:
why change it to `let`? just wondering

Reply:
Because at one point I mock a piece of it, which rubocop-rspec dislikes.